### ISSUE TITLE
Storing [object Object] in DynamoDB saveUnknown

### ISSUE THREAD

**Issue by fishcharlie** [dynamoosejs/dynamoose#669]
### Summary:

When trying to store a complicated object using saveUnknown, Dynamoose stores parts of the object as the string `[object Object]` instead of storing the underlying data.

#### Schema & Model
```js
const dynamoose = require("dynamoose");
const schema = new dynamoose.Schema({
	"id": {
		"type": String,
		"required": true
	}
}, {
	"saveUnknown": true
});
module.exports = dynamoose.model(`Wikidata`, schema, {});
```

#### General
```js
const data = (await axios.get(`https://www.wikidata.org/wiki/Special:EntityData/Q2.json`)).data.entities.Q2;
return Wikidata.create(data, {"overwrite": true});
```

### Current output and behavior:

Currently it is storing the claims as `[object Object]` as a string.

### Expected output and behavior:

The RAW DynamoDB data to not include `[object Object]` strings.

### Environment:

Operating System: macOS
Operating System Version: 10.15 Beta (19A536g)
Node.js version (`node -v`): v10.16.0
NPM version: (`npm -v`): 6.9.0
Dynamoose version: 1.11.0
Dynamoose Plugins: NA

### Dynamoose Plugins:

- [ ] Yes, I believe that one or multiple 3rd party Dynamoose plugins are effecting this issue
- [ ] No, I believe this issue is independent of any 3rd party Dynamoose plugins I'm using
- [ ] Unknown, I'm unsure if Dynamoose plugins are effecting this issue
- [x] I am not using any Dynamoose plugins

### Type (select 1):
- [x] Bug report
- [ ] Feature suggestion
- [ ] Question
- [ ] Other suggestion
- [ ] Something not listed here

### Other:
- [x] I have read through the Dynamoose documentation before posting this issue
- [x] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [x] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [x] I have tested the code provided and am confident it doesn't work as intended
- [x] I have ensured that all of my plugins that I'm using with Dynamoose are listed above
- [x] I have filled out all fields above
- [x] I am running the latest version of Dynamoose

**Comment by blakedietz** [comment]
@fishcharlie was this a verified bug? If so I may take a shot at it over the weekend to help pitch in.

**Comment by fishcharlie** [comment]
> @fishcharlie was this a verified bug? If so I may take a shot at it over the weekend to help pitch in.

@blakedietz Yeah, I didn't test the EXACT code I put in the sample code, but pretty confident that code will produce the same problem.

I didn't have time to write up automated tests for it to verify.

If you wanna take a look that'd be awesome!

**Comment by thetumper** [comment]
We have a scenario that results in this issue.  In our case, we have a field named "type" on our schema, which obviously is a reserved word.  Looking for a way to tell dynamoose "this is our 'type' not your reserved word 'type'".  Might be easiest for us to just rename our fields.

Maybe you're already aware, but your data from the URL in this report appears to have the same issue -- at least one data object has a field named "type".

**Comment by fishcharlie** [comment]
The goal is to fix this in the version 2.0 rewrite. As I mentioned in the PR #677 we are accepting submissions for automated tests to be added to the test suite to ensure these type of problems get fixed correctly. @thetumper If you want to look into possibly adding some automated tests to ensure that case gets covered, that'd be awesome!

If not, posting a MCVE example here would be great!
